### PR TITLE
New version: GTK 4.14.1

### PR DESCRIPTION
**Pull request by taozuhong**

### CODE REVIEW

File: mingw-w64-gtk4/PKGBUILD [hunk 1]
```diff
@@ -29,10 +29,11 @@ depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
          "${MINGW_PACKAGE_PREFIX}-json-glib"
          "${MINGW_PACKAGE_PREFIX}-libepoxy"
          "${MINGW_PACKAGE_PREFIX}-pango"
+         "${MINGW_PACKAGE_PREFIX}-shaderc"
```
Comment (lazka) [inline mingw-w64-gtk4/PKGBUILD:32]: also needs a dep on vulkan-loader

**Comment by lazka** [comment]
The vulkan backend isn't compatible with 32bit pointers (see https://gitlab.gnome.org/GNOME/gtk/-/issues/6033), so it needs to be disabled on 32bit arches.

Let me know if you need any help.

**Comment by lazka** [comment]
And unrelated to the build, it crashes right away here.

File: mingw-w64-gtk4/PKGBUILD (hunk 1 above)
Comment (taozuhong) [reply mingw-w64-gtk4/PKGBUILD:32]: The CI result shows that vulkan-loader is not required.

Comment (lazka) [reply mingw-w64-gtk4/PKGBUILD:32]: It links against vulkan-1.dll

and the clang CI has a broken dependency: `vulkan-1.dll => not found`

**Comment by lazka** [comment]
With 4.14.1 and newer pango it no longer crashes, but there are glitches and font rendering is broken:

image

**Comment by lb90** [comment]
Bisected to https://gitlab.gnome.org/GNOME/gtk/-/commit/b569470b87d143d30c2388046c9a6450696f19b8

**Comment by pbs3141** [comment]
What happens if you revert https://gitlab.gnome.org/GNOME/gtk/-/commit/b569470b87d143d30c2388046c9a6450696f19b8 as suggested above, and implement the icon fix from https://gitlab.gnome.org/GNOME/gtk/-/issues/6564#note_2059394? Then it might be possible to get this update out the door.

**Comment by lazka** [comment]
yeah, worth a shot

**Comment by lazka** [comment]
I'm getting lots of pango crashes with this, I've filed https://gitlab.gnome.org/GNOME/pango/-/issues/789

Maybe we'll have to revert.

**Comment by lazka** [comment]
Workaround in #20482

**Pull request merged**